App.UserSupport/UserSupportModule.cs

```diff
@@ -1,13 +1,54 @@
 ﻿using App.Configuration;
 using Castle.MicroKernel.Registration;
 using Castle.Windsor;
+using App.UserSupport.Models;
+using Microsoft.EntityFrameworkCore;
+using System.Collections.Generic;
 
 namespace App.UserSupport
 {
     public class UserSupportModule : IModule
     {
         public void Initialize(IWindsorContainer container)
-        {     
+        {
+            RegisterDbContext(container);
+        }
+        private Handling[] handlings = new Handling[5];
+
+        IEnumerable<Handling> InitHandlings()
+        {
+            handlings[0] = new Handling(new Client("Nikita"), "Some times i have troubles", 2,1);
```

Comment:
1) you hardcode ids for each entity. It`s quite boring, isn`t it ? :)
2) in "InitHandlings" method you initialize private field, but you also return it`s value, why ? Maybe there should be a single way of operating with this variable